tools/run_tests/run_xds_tests.py

```diff
@@ -69,10 +69,11 @@
     'header_matching',
     'circuit_breaking',
     'timeout',
+    'fault_injection',
 ]
```

Comment:
How do we enable this test case for each language?

Reply:
We have to change the scripts in the various repos to start including it.  E.g.

https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_xds_bazel_test_in_docker.sh
https://github.com/grpc/grpc-go/blob/master/test/kokoro/xds.sh
https://github.com/grpc/grpc-java/blob/master/buildscripts/kokoro/xds.sh
(etc)